fe1-web/src/features/evoting/network/ElectionHandler.ts

```diff
@@ -35,7 +37,7 @@ export function handleElectionSetupMessage(msg: ProcessableMessage): boolean {
   const storeState = getStore().getState();
   const lao = getCurrentLao(storeState);
   if (!lao) {
-    console.warn(makeErr('no LAO is currently active'));
+    console.warn(makeErr(STRINGS.no_active_lao));
     return false;
   }
 
```

Comment:
Github doesn't allow me to comment on the lines below (48, 60) but it seems that we're only looking for the `lao.id` and we probably shouldn't need the full `Lao` object.

Reply:
Seems like it yes and maybe not even that depending on whether we keep the laoId in the election messages. Also depends what we should see as the source of truth 🤔 For the election#open handler I now return false if the message laoId and the active laoId do not match. This should probably be added to the other handlers as well but I will do that in a seperate PR.

Reply:
All `ProcessableMessage`s  anyway contain a `.laoId` parameter which is derived from the channel on which they're posted. This is relatively new in the codebase, hence we may need neither the `lao` parameter in the election messages nor the check of the current active LAO. In principle, we could be connected to multiple LAOs (passively) but only one being currently active 😉